samples/02_commands_framework/Services/CommandHandlingService.cs

```diff
@@ -37,10 +41,18 @@ public async Task MessageReceivedAsync(SocketMessage rawMessage)
 
             // This value holds the offset where the prefix ends
             var argPos = 0;
+            // Perform prefix check. You may want to replace this with
+            // (!message.HasCharPrefix(_prefix, ref argPos))
```

Comment:
```suggestion
            // (!message.HasCharPrefix('!', ref argPos))
```
I think we should provide an example that matches the format "!help" as described on the next line.